pkg/windows/windowsupdate/session.go

```diff
@@ -55,7 +56,7 @@ func (iUpdateSession *IUpdateSession) GetLocal() (uint32, error) {
 
 func (iUpdateSession *IUpdateSession) SetLocal(locale uint32) error {
 	_, err := oleconv.ToUint32Err(oleutil.PutProperty(iUpdateSession.disp, "UserLocale", locale))
-	return errors.Wrap(err, "getproperty userlocale")
+	return fmt.Errorf("getproperty userlocale: %w", err)
```

Comment:
Optional -- should be `putproperty`?

Reply:
yea I think so, and it need a `err != nil` check, fixed